### PR TITLE
Fix empty lines hiding and a missing translation

### PR DESCRIPTION
**Pull request by jluukka**
Minor UI fixes:
- Translate the loading message for react-select
- Fix buggy empty line hiding logic

### CODE REVIEW

File: src/SearchModal/SearchModal.js [hunk 1]
```diff
@@ -227,19 +227,33 @@ class SearchModal extends Component {
...
               loadingText={localizationTexts.loading}
               noDataText={loading ? '' : localizationTexts.noData}
               loading={loading}
               pages={pages}
               page={page}
               onPageChange={this.handlePageChange}
               onPageSizeChange={this.handlePageSizeChange}
+              getTrGroupProps={
+                (state, row) => {
+                  console.log({ row });
```
Comment (shirleyqiantingliu) [inline src/SearchModal/SearchModal.js]: is this needed?

File: src/SearchModal/SearchModal.js [hunk 2]
```diff
@@ -227,19 +227,33 @@ class SearchModal extends Component {
...
+                }
+              }
               getTrProps={
-                (state, row) => ({
-                  onClick: () => this.handleSelectRow(row),
-                  className: selectedRow && row && selectedRow.index === row.index ? "selected" : ""
-                })
+                (state, row) => {
+                  const onClick = () => this.handleSelectRow(row);
+                  const className = selectedRow && row && selectedRow.index === row.index ? "selected" : "null";
```
Comment (shirleyqiantingliu) [inline src/SearchModal/SearchModal.js]: not familiar with this className. So if it is "null" it does not do anything?

File: src/SearchModal/SearchModal.js (hunk 1 above)
Comment (jluukka) [reply src/SearchModal/SearchModal.js]: It is not. Removing...

File: src/SearchModal/SearchModal.js (hunk 2 above)
Comment (jluukka) [reply src/SearchModal/SearchModal.js]: This was garbage left from development. Modifying "null" to an empty string.

**Pull request merged**